events/elections/README.md

```diff
@@ -21,24 +21,26 @@ eligibility for voting, eligibility for candidacy, maximal representation, etc.
 - Election period start
   - It takes time to create the poll in CIVS, so don’t give a specific hour, instead say “Morning of the 10th” or something vague.
 - Election period stop
-  - CIVS needs to be manually stopped, so an actual person needs to click for the poll to stop, so this needs to be a human friendly time.
+  - CIVS needs to be manually stopped, so an actual person needs to click for the poll to stop, so this needs to be a human friendly time
 - Results announcement date
 - Draft dates will then be passed to the Steering Committee for final approval
+- Once Steering approves the dates election officers will work with SIG Contributor Experience to put the election dates on the community calendar
+- Election officers coordinate with SIG Docs and appropriate subprojects to coordinate election results blog post
 
 ## Process
```

Comment:
Not a blocker, but we should add a release style timeline table. Example: [1.17 Timeline](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.17#timeline). Essentially a list of tasks or milestones by week 0, week 1.

I created a tracking issue with something similar for this past election: https://github.com/kubernetes/community/issues/4118